sdk/docs/core/README.md

```diff
@@ -85,13 +85,25 @@ As our SDK performs operations, it can send log messages to a customer-defined c
 To enable logging, you must first write a callback function that our logging mechanism will invoke periodically with messages. The function signature must match this type definition (defined in the [az_log.h](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/inc/azure/core/az_log.h) file):
 
    ```C
-   typedef void (*az_log_fn)(az_log_classification classification, az_span message);
+   typedef void (*az_log_message_fn)(az_log_classification classification, az_span message);
```

Comment:
This rename should go to changelog.

Reply:
This rename didn't occur in any recent release. It was done quite a while back, well before preview 4. It doesn't make sense to add an entry to the changelog for a change that wasn't made in that release. All this is doing is caching the docs up to what already changed.